packages/swingset-runner/bin/graphDisk

```diff
@@ -0,0 +1 @@
+mainLauncher
```

Comment:
Could we maybe use a pattern where this says `mainLauncher graphDisk` instead of having `mainLauncher` look at `process.argv[1]` to sense the name under which it was invoked? This technique would get confused if, say, I made a symlink to `graphDisk` and ran the tool through my symlink, without knowing that it was internally paying attention to the name of the original invocation.

Reply:
Hmm.  I like that; it's clearer AND less brittle.

Reply:
No, I don't know what I was thinking when I wrote that last comment.  You're being fooled by how GitHub displays the file diff.  There's no text in the file that says `mainLauncher` that could be changed to instead say `mainLauncher graphDisk`.  The `graphDisk` file is a symlink to `mainLauncher`.

Reply:
oh, ok, got it. yeah, github displays symlinks weirdly.

But actually, that's kind of worse, you couldn't even clone the repo on windows in that case. Let's make it a rule to not put symlinks into version control.

But fortunately I think the plan we came up with on the phone should work better.